tools/site_admin/copy_pages.php

```diff
@@ -519,13 +519,13 @@ function do_stuff( $projectid_, $from_image_, $page_name_handling,
     echo "</li>\n";
 
     echo "<li>\n";
-    if ($merge_wordcheck_files) 
+    if ($merge_wordcheck_data)
     {
-        echo _("The WordCheck files from the source project WILL be merged into the destination project");
+        echo _("The WordCheck data from the source project WILL be merged into the destination project");
     }
     else
     {
-        echo _("The WordCheck files WILL NOT be merged");
+        echo _("The WordCheck data WILL NOT be merged");
```

Comment:
Above, for the button, you have the text: `Merge WordCheck files and events into destination project`. It might be good to use that same wording for thest two messages.

Reply:
Good idea - fixed.